package.json

```diff
@@ -57,11 +57,11 @@
     "release": "ts-node --transpile-only automation/run.ts release",
     "pretest": "npm run build",
     "test": "npm run test:source && npm run test:standalone",
-    "test:source": "cross-env BALENA_CLI_TEST_TYPE=source mocha --timeout 6000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
+    "test:source": "cross-env BALENA_CLI_TEST_TYPE=source mocha --timeout 12000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
```

Comment:
Increasing the timeout from 6s to 12s in a separate commit. I don't know if this will actually solve the relatively frequent timeout errors in CI builds in recent PRs; I though it was worth trying.